Automattic/wp-calypso
# Stepper: Make intro step generic and use it in Newsletters flow

#### Proposed Changes

* This makes the intro step generic and reusable in the newsletters flow. 
* It also fixes the translation issue (the intro message was split into two phrases).

#### Testing Instructions

1. Go to [/setup/intro?flow=newsletters](https://container-flamboyant-hugle.calypso.live/setup/intro?flow=newsletters), should look like the design.
2. Go to [/setup/intro?flow=link-in-bio](https://container-flamboyant-hugle.calypso.live/setup/intro?flow=link-in-bio), should look like the design.

#### Screenshots for translators
![calypso localhost_3000_setup_intro_flow=link-in-bio](https://user-images.githubusercontent.com/17054134/182175716-e34f9a17-c64c-46d8-a15e-f9b84c047e4c.png)
![calypso localhost_3000_setup_intro_flow=newsletters](https://user-images.githubusercontent.com/17054134/182175799-8921dbd6-d25b-4013-a432-aa7ed5e9674e.png)
